dipper/sources/ZFIN.py

```diff
@@ -1247,16 +1134,15 @@ def _process_g2p(self, limit=None):
         with open(raw, 'r', encoding="utf8") as csvfile:
             filereader = csv.reader(csvfile, delimiter='\t', quotechar='\"')
             for row in filereader:
-                line_counter += 1
-
+                line_counter += 1 
+              
```

Comment:
trailing spaces  will cause incessant nagging and annoying diffs

Reply:
Okay, sorry about that, pylint says no trailing whitespace now